src/frontend/admin/src/components/templates/products/form/sections/target-courses/ProductFormTargetCourseRow.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from "react";
-import { useIntl } from "react-intl";
+import { defineMessages, useIntl } from "react-intl";
+import GppGoodIcon from "@mui/icons-material/GppGood";
```

Comment:
I'll use the school icon WDYT ?
`import SchoolIcon from '@mui/icons-material/School';`
or grading ?

`import GradingIcon from '@mui/icons-material/Grading';`